### DISCUSSION TITLE
I can't confirm that I'm a student

### DISCUSSION THREAD

**Discussion by eshevlyakova** [community/community#145062] [GitHub Education]
### Hi everyone,

I enclosed the original student card, certificate of study at uni. I have also attached a translation+original student card, certificate of study and still get a refusal for the reason:
‘The image you selected does not appear to contain your school name. Your complete school name must appear in your document, not only the school logo. You may include multiple documents together. If your official document is not in English then you may photograph the original next to an English translation.’

**Comment by davevad93** [comment]
Hi @eshevlyakova ,

Make sure the academic proof you submit contains your full name, school name and a current date in a `DD/MM/YYYY`, `MM/DD/YYYY` or `YYYY/MM/DD` format. If you encounter further issues, you may submit alternative documents that prove your current student status such as your class schedule, transcript, and affiliation or enrollment verification letter.

Also check out the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352).

**Comment by eshevlyakova** [comment]
I checked, the name matches, there's the name of the school and the date


<img width="1002" alt="Screenshot 2024-11-19 at 22 58 48" src="https://github.com/user-attachments/assets/74bed484-f616-45a8-b73b-d4ae50ab0a98">

**Reply by davevad93** [reply]
Hey @eshevlyakova , you're right, your academic proof contains all the necessary information and your request should be accepted, but somehow the automated process doesn't recognize it correctly, try to submit another proof (for example your class schedule or your last exam results), sometimes submitting a second document solves the problem. If even after submitting additional evidence your application is still rejected, try to contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain the problem you're experiencing with the application attaching also the screenshots of the academic documentation you've submitted.
